lib/gli/command_finder.rb

```diff
@@ -9,16 +9,16 @@ class CommandFinder
 
     def initialize(commands, options = {})
       self.options = DEFAULT_OPTIONS.merge(options)
-      self.commands_with_aliases = expand_with_aliases(commands)
+      @commands_with_aliases = expand_with_aliases(commands)
```

Comment:
What warning does this shut off?  Hard to see how this is an improvement (it's not a *de*-improvement either, just not clear what this buys us)